application/src/main/java/org/togetherjava/tjbot/commands/moderation/ReportCommand.java

```diff
@@ -79,6 +77,8 @@ private Cache<Long, Instant> createCooldownCache() {
     @Override
     public void onMessageContext(MessageContextInteractionEvent event) {
         long userID = event.getUser().getIdLong();
+        String reportedMessageTimestamp =
```

Comment:
afaik `getTimeCreated()` returns `OffsetDateTime` (bc JDA/Discord is stupid), no? If thats the case, please first go to instant. `getTimeCreated().toInstant()`.

that way ull make sure the string is a correct instant in all cases